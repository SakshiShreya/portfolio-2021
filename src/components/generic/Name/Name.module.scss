@import "../../../utilities/partials/colorCodes";

.orange {
  color: map-get($map: $orange, $key: "normal");
}

.green {
  color: map-get($map: $green, $key: "normal");
}