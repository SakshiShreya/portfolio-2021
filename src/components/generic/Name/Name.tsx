import React from "react";
import styles from "./Name.module.scss";

interface NameProps {}

const Name: React.FunctionComponent<NameProps> = () => {
  return (
    <>
      <span data-testid="green" className={styles.green}>
        S
      </span>
      akshi{" "}
      <span data-testid="orange" className={styles.orange}>
        S
      </span>
      hreya
    </>
  );
};

export default Name;
