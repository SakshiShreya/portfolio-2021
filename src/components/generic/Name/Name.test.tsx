import renderer from "react-test-renderer";
import Name from "./Name";

it("should render Name", () => {
  const elem = renderer.create(<Name />).toJSON();
  expect(elem).toMatchSnapshot();
});

// TODO: Name should have green and orange class
