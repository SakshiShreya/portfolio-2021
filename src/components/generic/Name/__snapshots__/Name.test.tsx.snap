// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Name 1`] = `
Array [
  <span
    className="green"
    data-testid="green"
  >
    S
  </span>,
  "akshi",
  " ",
  <span
    className="orange"
    data-testid="orange"
  >
    S
  </span>,
  "hreya",
]
`;
